app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -263,32 +261,29 @@ export default {
     },
   },
 
-  created() {
-    bus.$on(BUS_EVENTS.SCROLL_TO_MESSAGE, this.onScrollToMessage);
-    bus.$on(BUS_EVENTS.SET_TWEET_REPLY, this.setSelectedTweet);
+  updated() {
+    this.$nextTick(() => this.scrollToEndOfMessages());
   },
 
-  mounted() {
-    this.addScrollListener();
+  created() {
+    bus.$on(BUS_EVENTS.SET_TWEET_REPLY, this.setSelectedTweet);
   },
 
   beforeDestroy() {
     this.removeBusListeners();
-    this.removeScrollListener();
   },
 
   methods: {
+    scrollToEndOfMessages() {
```

Comment:
There would be couple of side effects with this. Can you cross check the current behaviour with

1. When agent is reading a message and visitor replies or a new activity comes in, it would scroll automatically to end interfering the agents reading experience 

2. When is conversation is scrolled up to load previous message, would it come to end ?